src/components/cats/index.js

```diff
@@ -8,6 +8,11 @@ export default props => (
 
     <p>{props.currentCat.description}</p>
 
+    <img src={props.currentAdopt.img} width="50%" height="50%" />
```

Comment:
So we want to display all of the cats that are in our array in redux here, not just one cat